### DISCUSSION TITLE
Is it possible to duplicate the layer manager?

### DISCUSSION THREAD

**Discussion by kamalesh-d** [GrapesJS/grapesjs#5025] [Q&A] [answered]
Hello, 

I am trying to create mirror effect of the sidepanel. (Same side panel displayed on both left and right side, so that user can toggle certain tabs in left vs right). I am achieving this through vuejs, by reusing the sidepanel component. However, my layer manager is not visible on both the sides. I am using the default layer manager (not programmatically). 

I tried change in config of layermanager by using, appendTo: ['#leftPanelLayer', '#rightPanelLayer'], or appendTo the class name. However, it always exist like Layer manager in left panel or right panel. 

Can I please know if it is possible to duplicate the layer manager? Please let me know the guidance to achieve it. 

Thanks, 
Kamalesh D.

**Comment by artf** [comment, marked as answer]
The default one can't be duplicated, in this case you have to [use a custom one](https://grapesjs.com/docs/modules/Layers.html#customization).
